### DISCUSSION TITLE
How do you rebuild Wazuh and osquery packages to include updated SSL cert?

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#10039] [Unsupported Versions]
I created a LetsEncrypt SSL cert for my manager and am having an issue communicating with launcher apparently due to a cert mismatch.  How do you rebuild the agent packages to include the new cert?

This is part of an earlier unresolved issue I opened almost two months ago (https://github.com/Security-Onion-Solutions/securityonion/discussions/9731).  The error thrown is:

transport: authentication handshake failed: x509: certificate signed by unknown authority

I was asked to provide the output of 'sudo salt-call pillar.get global' which looks like this:

```
# salt-call pillar.get global
local:
    ----------
    airgap:
        False
    dockernet:
        172.17.0.0
    fleet_custom_hostname:
        None
    fleet_hostname:
        so-manager.seceng.mydomain.com
    fleet_ip:
        10.0.210.159
    fleet_manager:
        True
    fleet_node:
        False
    fleet_packages-timestamp:
        2023-03-01-20:30
    fleet_packages-version:
        5
    hnmanager:
        10.0.0.0/8,192.168.0.0/16,172.16.0.0/12
    ids:
        Suricata
    imagerepo:
        security-onion-solutions
    managerip:
        10.0.210.159
    mdengine:
        ZEEK
    pipeline:
        redis
    sensoronikey:
        weSsnQnA7D11YaMuHBKL
    soversion:
        2.3.220
    url_base:
        so-manager.seceng.mydomain.com
    wazuh:
        1
```

I'd love to get this resolved and the agents deployed.  Right now, I've had to disable the 'launcher.so-launcher' service on all of my nodes due the sheer volume of the above error so if anyone can point me in the right direction, I would be very appreciative! 

Thank you!

**Comment by defensivedepth** [comment]
What does the launcher flag file look like?
